packages/sdk/src/next/types/PageTypes.ts

```diff
@@ -0,0 +1,30 @@
+export type Page = {
```

Comment:
Does this already need to be here?

Reply:
I removed id from there, since it was a wrong assumption

Reply:
This one also contains the visible property